customresources/s3writer/__init__.py

```diff
@@ -0,0 +1 @@
+#
```

Comment:
Out of curiosity, why not just a blank file?
Hopefully not a 
`# this file intentionally left blank` 
situation.